client/app/account/api/changePasswordApi.ts

```diff
@@ -0,0 +1,10 @@
+import apiClient from '@/_api/base'
+import { UserType } from '@/_types'
+import { PasswordSchema } from '../schema'
+
+export const changePasswordApi = async (
+  data: PasswordSchema,
+): Promise<UserType> => {
+  const response = await apiClient.put<UserType>('/me/change-password', data)
```

Comment:
Return type is `{ message: string }`  if you return like this `return { message: 'Password updated successfully.' };`

Reply:
Got it! 
c852d0de6de152c6b0300c576333c89bd95dc47b